giraffe/src/utils/formatters.ts

```diff
@@ -139,6 +139,7 @@ export const timeFormatter = ({
   // for example, modifying the timezone before determing the `am/pm` will
   // output the timezone incorrectly. The same goes for determining the `HH`, etc...
   const formatStringFormatter = createDateFormatter({
+    hh: ({hour}) => (Number(hour) < 10 ? `0${hour}` : String(hour)),
     HH: ({lhour}) => {
       if (format && format.includes('a')) {
```

Comment:
this might not be necessary now that we've added in the other formatter data

Reply:
No, let's keep it for now. There are very subtle differences between the two and there might be a use case for this when we build out the options in time formats.